src/ps3d.f90

```diff
@@ -80,15 +80,26 @@ subroutine pre_run
             ! calculate the initial \xi and \eta mean and save it in ini_vor_mean:
             ini_vor_mean = calc_vorticity_mean()
 
+            call calculate_peref
+
             call vor2vel
 #ifdef ENABLE_BUOYANCY
             bbdif = maxval(buoy) - minval(buoy)
```

Comment:
Do we still use bbdif?  I thought we use instead the generalised enstrophy including the integral of |gradb|?

Reply:
We use bbdif for molecular viscosity only.

Reply:
OK, this is fine